omod/pom.xml

```diff
@@ -76,6 +76,11 @@
 			<version>${appuiVersion}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.github.ben-manes.caffeine</groupId>
+			<artifactId>caffeine</artifactId>
+			<version>2.8.2</version>
+		</dependency>
```

Comment:
This should have `<scope>provided</scope>`... if we even need this dependency. It should be provided by the FHIR2 module.

Reply:
Thanks, I'll change it.